### DISCUSSION TITLE
Word to PDF Font not correct

### DISCUSSION THREAD

**Discussion by Hutch79** [gotenberg/gotenberg#1020] [Q&A] [answered]
Since Version 8.11.1 the `/forms/libreoffice/convert` endpoint does not use the Font provided by the Word document but instead changes the font to another one.
I'd guess this could be caused by the update from LibreOffice 24.2.4 to 24.8.2 which was done in Gotenberg 8.11.1.

I'm not sure if this is something in scope of Gotenberg to file an issue, therefore I started it as a discussion.
I tried to search in the LibreOffice Bug tracker for this issue but honestly, I have no clue how this thing works ^^'

**Comment by Hutch79** [comment]
I noticed that the error was not necessarily with Gotenberg/LibreOffice.
I used Aptos as font and when I changed it to Calibri it worked again.

My best guess is that Aptos is no longer in LibreOffice, and therefore it used a fallback font.

Sorry for the unnecessary discussion.

**Reply by gulien** [reply]
Thanks for the feedback @Hutch79!

I did not change the installed fonts, so it's weird that a font previously available stopped working 🤔 Maybe it's actually the fallback font that changed, and it never worked in the first place?

**Reply by Hutch79** [reply]
Could be, although I don't know how I could check that.

I'd guess from your answer that the installed fonts in Gotenberg are not determent by LibreOffice, but by Gotenberg itself?

**Reply by gulien** [reply]
LibreOffice is using the installed font in the system. I install in the image the ones recommended by LO plus a bunch of others, but the list did not change between the two releases. 

You can try to build your own Docker image and install the Aptos font: https://gotenberg.dev/docs/configuration

**Reply by Hutch79** [reply]
Ok , good to know.
Am I right in the assumption that the run statement on this line installs the fonts in Gotenberg?
https://github.com/gotenberg/gotenberg/blob/main/build/Dockerfile#L92

It installs a tool to get MS fonts and also installs a couple of fonts directly from the deb repo and the Google fonts repo.
But I don't see which MS fonts are installed.

Is there a list where I can see that?

**Reply by gulien** [reply, marked as answer]
Yep that's the statement 😄 

AFAIK, here is the list of MS fonts:

Andale Mono
Arial Black
Arial (Bold, Italic, Bold Italic)
Comic Sans MS (Bold)
Courier New (Bold, Italic, Bold Italic)
Georgia (Bold, Italic, Bold Italic)
Impact
Times New Roman (Bold, Italic, Bold Italic)
Trebuchet (Bold, Italic, Bold Italic)
Verdana (Bold, Italic, Bold Italic)
Webdings

**Reply by Hutch79** [reply]
Great, thanks for the info and also for your work on the Project!

Well, we're changing to one of the Noto fonts instead of Aptos.
You were right about the fallback font, it never was Aptos from the start. ^^'

**Reply by gulien** [reply]
Great! Happy to help 👍
